portfolio/src/main/webapp/script.js

```diff
@@ -63,4 +63,13 @@ function getComments() {
 window.onload = function() {
   randQuote();
   getComments();
-}
\ No newline at end of file
+}
+
+// Listens for Comments section button
+window.addEventListener("load", function() {
```

Comment:
Is there any reason you didn't add the logic in this function directly to window's onload attribute (along with `randQuote` and `getComments`?

Reply:
I forgot to optimize that implementation in this PR. The next PR should have ```randQuote``` and ```getComments``` in the same block.